packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -44,12 +44,15 @@ class FlutterDevice {
     _viewsCache = null;
   }
 
-  void connect() {
+  /// If the [reloadSources] parameter is not null the 'reloadSources' service
+  /// will be registered
```

Comment:
I've added a more detailed description of the process and of the reasons around it.

done